build_cpp.sh

```diff
@@ -0,0 +1,11 @@
+mkdir cpp/build
+pushd cpp/build
+python3 -m venv ENV
+source ENV/bin/activate
+pip3 install numpy cython pandas
+export ARROW_HOME=$(pwd)/cpp/build/arrow/install
+echo "ARROW HOME SET :"${ARROW_HOME}
```

Comment:
improve the scripts to get the build location as an argument, without assuming cpp/build directory, if not specified we can take cpp/build as default

Reply:
added it as a constant field